components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -424,6 +424,44 @@ public void revokeOAuthConsentByApplicationAndUser(String username, String tenan
         }
     }
 
+    /**
+     * Revoke the OAuth Consent which is recorded in the IDN_OPENID_USER_RPS table against the Application
+     *
+     * @param applicationName - Name of the OAuth App
+     * @throws IdentityOAuth2Exception - If an unexpected error occurs.
+     */
+    @Override
+    public void revokeOAuthConsentByApplication(String applicationName) throws IdentityOAuth2Exception {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Revoking OAuth consent for application: " + applicationName);
+        }
+
+        if (applicationName == null) {
+            log.error("Could not remove user consents for application " + applicationName);
```

Comment:
here the applicationName will be null. hence this log will be confusing

Reply:
Addressed in [40fde89](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/40fde89cb099493c8f2a302b34b1141d9aea3e8d)